README.md

```diff
@@ -133,16 +133,35 @@ Ubuntu 16.04     | no
 Ubuntu 18.04     | yes
 
 
-## Localization
+## Localization and translation
+
+Zonemaster comes with translation for the following
+languages
+
+* English (en, en_US.UTF-8)
+* French (fr, fr_FR.UTF-8)
+* Swedish (sv, sv_SE.UTF-8)
+* Danish (da, da_DK.UTF-8)
+
+The support of localization or translation is however not complete due to
+complexity in the implementation of `gettext` functions in different OSs.
```

Comment:
Should we really be shifting blame here? IMHO it would be better to just state what works and what doesn't. Maybe with a link to some issue in case anyone is interested.

Reply:
I do not understand what you mean by "shifting blame". Do you want that specific sentence to be removed and replaced by a link to an issue containing what?

Reply:
That's what I had in mind.

Reply:
I can remove that sentence, but I have no issue to link to. It is OK.

Reply:
Updated.